include/spirv/unified1/spirv.core.grammar.json

```diff
@@ -6698,6 +6698,768 @@
       "capabilities" : [ "SubgroupAvcMotionEstimationINTEL" ],
       "version" : "None"
     },
+    {
+      "opname" : "OpArbitraryFloatSinCosPiINTEL",
+      "class"  : "@exclude",
+      "opcode" : 5840,
+      "operands" : [
+        { "kind" : "IdResultType" },
+        { "kind" : "IdResult" },
+        { "kind" : "IdRef", "name" : "'A'" },
+        { "kind" : "LiteralInteger", "name" : "'M1'" },
+        { "kind" : "LiteralInteger", "name" : "'Mout'" },
+        { "kind" : "LiteralInteger", "name" : "'EnableSubnormals'" },
+        { "kind" : "LiteralInteger", "name" : "'RoundingMode'" },
+        { "kind" : "LiteralInteger", "name" : "'RoundingAccuracy'" }
+      ],
+      "capabilities" : [ "ArbitraryPrecisionFloatingPointINTEL" ],
+      "extensions" : [ "SPV_INTEL_arbitrary_precision_floating_point" ],
```

Comment:
I recommend that things that are not capabilities should normally not have an extensions list.
Enable them by a capabilities list, and then have the capabilities enabled by the extensions list.
That makes it less confusing because then there is no question about whether it's (capabilties *and* extensions) vs. (capabilities *or* extensions).